lib/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const OZONE_SERVICE_DID =
+  process.env.NEXT_PUBLIC_OZONE_SERVICE_DID ||
+  'did:plc:ar7c4by46qjdydhdevvrndac'
```

Comment:
I am not sure that we need to hardcode our labeler in Ozone.  The service DID may also be set at runtime by fetching `ozone-metadata.json`, as is the case when running the dockerized Ozone distribution.

Reply:
ooops... accidental commit, will clean up

Reply:
@foysalit I think I saw this get cleaned-up, but it still is kicking around here.

Reply:
yh idk what's up but I think I lost some changes when merging main into this.